pkg/sfu/buffer/rtpstats_receiver.go

```diff
@@ -154,7 +154,7 @@ func (r *RTPStatsReceiver) Update(
 				return
 			}
 		}
-		if -gapSN >= cNumSequenceNumbers {
+		if -gapSN >= cNumSequenceNumbers/2 {
```

Comment:
Seeing some largish jumps in very rare cases in some other log (i. e. when we receive Receiver Report) but this is not logging. So, reducing the threshold here to check if this hits.